tests/sympc/tensor/replicatedshare_tensor_test.py

```diff
@@ -1,6 +1,43 @@
+# third party
+import torch
+
+from sympc.session import Session
+from sympc.session import SessionManager
 from sympc.tensor import ReplicatedSharedTensor
 
 
-def test_import_RSTensor():
+def test_import_RSTensor() -> None:
 
     ReplicatedSharedTensor()
+
+
+def test_hook_method(get_clients) -> None:
+    alice, bob = get_clients(2)
```

Comment:
Q: We are implementing RST mainly for FALCON don't you think we should check it for 3 parties?
After all, Falcon is A 3-Party Machine Learning Service. :D

Reply:
Yeah you are right :slightly_smiling_face:,I thought to include the sanity check in protocol in later commits.